packages/core/lib/modules/accounts.js

```diff
@@ -78,9 +78,11 @@ export class Accounts {
       true,
     );
 
-    currency.value = balance;
+    if (currency.value !== balance) {
```

Comment:
what was happening here?

Reply:
It could potentially create some loop, since every document update will push an event and there are hooks listening for updates which could potentially try to fetch balances. Just trying to avoid this to happen. It's also good for performance.